CMakeLists.txt

```diff
@@ -56,297 +63,80 @@ zephyr_include_directories(
   include
   include/drivers
   ${PROJECT_BINARY_DIR}/include/generated
-  ${USERINCLUDE}
-  ${STDINCLUDE}
+  ${USERINCLUDE}  # unused
+  ${STDINCLUDE}   # unused
```

Comment:
Let's either drop these, or document them as user-settable.

I'd suggest dropping them.

I'd also suggest doing this in a separate PR.

Reply:
@SebastianBoe agreed, I can do that in a follow-up PR.

Reply:
In general it is better to do it in a preceeding PR, as that keeps PR's small. This is especially true for big PR's like this one.